### PR TITLE
Add newsletter section styles

### LINKED ISSUE #581: Improve the styles of the news letter subscription form
**Is your feature request related to a problem? Please describe.**
- We need to improve the styles of the subscription form

**Describe the solution you'd like**
- Choose a suitable design and improve the style

### PR DESCRIPTION
**Pull request by Madhawa97**
## Purpose

The purpose of this PR is to fix #581 

## Goals

To improve the styles of the subscription form

## Approach

Choose a suitable design and improve the style

### Screenshots
Screenshot 2022-12-14 at 18 43 03

## Related PRs
 
None

### CODE REVIEW

**Comment by anjula-sack** [comment]
Thank you @Madhawa97 Shall we use black as the bg colour and do the styling? So we can make it look like a part of the footer/

**Comment by Madhawa97** [comment]
Sure @anjula-sack.

**Comment by Madhawa97** [comment]
Update: 

Screenshot 2022-12-14 at 19 11 42

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: donor-portal/src/components/SubscriptionForm/styles.css [hunk 1]
```diff
@@ -1,6 +1,71 @@
...
+    gap: 0rem;
+  }
 }
+
+
+@media only screen and (min-width: 1024px) {
+  .form-container h2 {
+    font-size: 32px;
+  }
+}
```
[the remark this answers is not in the document]
Comment (Madhawa97) [reply donor-portal/src/components/SubscriptionForm/styles.css]: Done @anjula-sack

File: donor-portal/src/components/SubscriptionForm/index.tsx [hunk 2]
```diff
@@ -10,31 +10,35 @@ export default function SubscriptionForm() {
...
+          />
           <input
-            type="email"
-            name="EMAIL"
-            className="required email"
-            id="mce-EMAIL"
-            required
+            type="text"
+            name="LNAME"
+            className=""
```
Comment (anjula-sack) [inline donor-portal/src/components/SubscriptionForm/index.tsx:24]:
```suggestion
```

**Review by anjula-sack — APPROVED** [review]
Good job! @Madhawa97

**Comment by Madhawa97** [comment]
Thank you @anjula-sack

**Pull request merged**